<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="20.382" tests="6" failures="2">
  <testsuite name="Root Suite" timestamp="2025-05-09T20:22:02" tests="0" file="cypress/e2e/features/assignment_submission.feature" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Assignment Submission" timestamp="2025-05-09T20:22:06" tests="6" time="16.176" failures="2">
    <testcase name="Assignment Submission Successfully submitting an assignment before deadline" time="0.000" classname="Successfully submitting an assignment before deadline">
      <failure message="Timed out retrying after 6000ms: expected &apos;http://localhost:3001/login&apos; to not include &apos;/login&apos;" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 6000ms: expected 'http://localhost:3001/login' to not include '/login'
    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/support/e2e.ts:13365:14)]]></failure>
    </testcase>
    <testcase name="Assignment Submission &quot;after each&quot; hook for &quot;Saving a draft submission&quot;" time="0.000" classname="&quot;after each&quot; hook for &quot;Saving a draft submission&quot;">
      <failure message="Cannot read properties of undefined (reading &apos;seconds&apos;)

Because this error occurred during a `after each` hook we are skipping all of the remaining tests." type="TypeError"><![CDATA[TypeError: Cannot read properties of undefined (reading 'seconds')

Because this error occurred during a `after each` hook we are skipping all of the remaining tests.
    at duration (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/assignment_submission.feature:9751:40)
    at Context.eval (http://localhost:3001/__cypress/tests?p=cypress/e2e/features/assignment_submission.feature:10124:65)]]></failure>
    </testcase>
  </testsuite>
</testsuites>